tracee-ebpf/tracee/co-re/struct_flavors.h

```diff
@@ -3,30 +3,68 @@
 
 #pragma clang attribute push (__attribute__((preserve_access_index)), apply_to = record)
 
+#include <vmlinux.h> // linting purposes
+
+// (struct kernfs_node *)->id was union kernfs_node_id before 5.5
+
 union kernfs_node_id {
     struct {
-        u32     ino;
-        u32     generation;
+	u32 ino;
+	u32 generation;
```

Comment:
Please use spaces instead of tabs

Reply:
sorry, vimrc didnt have the new filenames set.